### PR TITLE
Loading Screen Overhaul

### PR DESCRIPTION
**Pull request by helloruhe**
## Changelog:
- Overhauled all current loading screens to match WoW Style
- Added 2 New loading screens

## Developer changelog:
- Replaced the 'council' loading screen image because of image perms
- A link to the loading screens psds and the template is in gdrive, the link is in the artist resource folder. was too big to upload here

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
- Open Game 
- If you can, look at the PSDs or images in game and let me know if it looks weird or are their any artifacts. The only one I think ist weird is the one with archimonde and the tree
- Could use feedback in general
# TODO
- [x] Add a few more loading screens (maybe 2+ more)
- [x] Warcrafish Flavor loading tips

### CODE REVIEW

**Review by ElMariuso — APPROVED** [review]
Better, I think. LGTM but wait for more reviews if possible

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:14]: LOADING_TIP_12:0 "Humans, Night Elves, Dwarves, Gnomes, Draenei, and the savage Worgen make up the illustrious Alliance. Proud and noble, courageous and wise, these races work together to preserve order in Azeroth. The Alliance is driven by honor and tradition. Its rulers are champions of justice, hope, knowledge and faith.\n\nIn a time when chaos and uncertainty reign, the Alliance remains steadfast in its determination to bring light to the darkest corners of the world."

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:15]: LOADING_TIP_13:0 "The Horde is made up of Orcs, Forsaken, Tauren, Trolls, Blood Elves, and most recently, Goblins. Misunderstood and cast aside, these diverse and powerful races strive to overcome their differences and unite as one in order to win freedom for their people and prosper in a land that has come to hate them.\n\nIn the Horde value action and strength above diplomacy and its leaders earn respect by the blade; wasting no time with politics. The brutality of the Horde's champions is focused, giving a voice to those who fight for it's survival."

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:16]: LOADING_TIP_14:0 "#T Did you know?#!\n\nGul'dan was banished from his village because he was born weak, crippled and deformed. He later slaughtered everyone in the village - even those who showed him kindness."

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:22]: LOADING_TIP_20:0 "#T Did you know?#!\n\nDeathwing can easily be recognized by his mandible and the colossal blade on the end of his gigantic tail."

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:25]: LOADING_TIP_23:0 "#T Did you know?#!\n\nKael'thas Sunstrider is in love with the mage Jaina Proudmoore. She does not seem to return the feeling."

File: localization/english/load_tips/load_tips_l_english.yml
Comment (Raged50) [inline localization/english/load_tips/load_tips_l_english.yml:75]: LOADING_TIP_73:0 "#T Did you know?#!\n\nIn the main building of Mistfall Village, a bush chicken is playing a troll guitar in the hopes of spare change."

**Review by Raged50 — APPROVED** [review]
LGTM, just that one image needs modifying and some minor changes

File: localization/english/load_tips/load_tips_l_english.yml
Comment (helloruhe) [reply localization/english/load_tips/load_tips_l_english.yml:75]: thats u

**Pull request merged**
